src/bindings/python/src/pyopenvino/graph/node.cpp

```diff
@@ -216,6 +217,58 @@ void regclass_graph_Node(py::module m) {
                 :return: Number of inputs.
                 :rtype: int
              )");
+    node.def("get_input_element_type",
+             &ov::Node::get_input_element_type,
+             py::arg("index"), 
+             R"(
+                Returns the element type for input i
+
+                :param index: Index of the input.
+                :type index: int
+                :return: Type of the input i
+                :rtype: openvino.runtime.Type
+             )");
+    node.def("get_input_partial_shape",
+             &ov::Node::get_input_partial_shape,
+             py::arg("index"),
+             R"(
+                Returns the partial shape for input i
+
+                :param index: Index of the input.
+                :type index: int
+                :return: PartialShape of the input i
+                :rtype: openvino.runtime.PartialShape
+             )");
+    node.def("get_input_shape",
+             &ov::Node::get_input_shape,
+             py::arg("index"),
+             R"(
+                Returns the shape for input i
```

Comment:
```suggestion
                Returns the shape for input i
```
or `index` ?

Reply:
yes, good idea. The whole Node doc was updated